x-pack/plugin/esql/src/main/generated/org/elasticsearch/xpack/esql/expression/function/scalar/ip/IpPrefixEvaluator.java

```diff
@@ -42,12 +42,12 @@ public IpPrefixEvaluator(Source source, EvalOperator.ExpressionEvaluator ip,
       EvalOperator.ExpressionEvaluator prefixLengthV4,
       EvalOperator.ExpressionEvaluator prefixLengthV6, BytesRef scratch,
       DriverContext driverContext) {
-    this.warnings = new Warnings(source);
     this.ip = ip;
     this.prefixLengthV4 = prefixLengthV4;
     this.prefixLengthV6 = prefixLengthV6;
     this.scratch = scratch;
     this.driverContext = driverContext;
+    this.warnings = Warnings.createWarnings(driverContext.warningsMode(), source);
```

Comment:
Is this change related somehow?

Reply:
that was a timing issue with my rebase. fixed